pkg/kubelet/kubelet_server_journal.go

```diff
@@ -142,6 +143,39 @@ func (a *journalArgs) Args() []string {
 	return args
 }
 
+// windowsArgs returns the PowerShell and Get-WinEvent cmdlet arguments for the given args.
+func (a *journalArgs) windowsArgs() []string {
+	args := []string{
+		"-c",
```

Comment:
NTS: This may additionally need to include: `-NonInteractive -ExecutionPolicy Bypass`